client/components/wp-components-wrapped/make-wrapped-component.tsx

```diff
@@ -17,9 +17,29 @@ export const makeWrappedComponent = <
 ) =>
 	React.forwardRef<
 		any,
-		ComponentProps< T > & { useBundledComponent?: boolean }
+		ComponentProps< T > & {
+			useBundledComponent?: boolean;
```

Comment:
With your cleanup, are there existing usages of `useBundledComponent` in the codebase, or are we planning on using it?
I wonder if we can just remove it 🤷